src/govuk/components/back-link/_index.scss

```diff
@@ -1,16 +1,19 @@
 @include govuk-exports("govuk/component/back-link") {
+  // Component font-size on the Frontend (used for calculations)
+  $font-scale-size: 16;
+  $font-size: $font-scale-size * 1px;
```

Comment:
I think we can go "unitless" with `govuk-em(7, 16)` if that's nicer?

Reply:
Good catch. `gov-em` internally converts it to a pixel value anyway, so saves us a step here, I guess!